pallets/account-linker/src/lib.rs

```diff
@@ -162,7 +162,7 @@ pub mod pallet {
 
 			let _ = ensure_signed(origin)?;
 
-			let current_block_number = <frame_system::Module<T>>::block_number();
+			let current_block_number = <frame_system::Pallet<T>>::block_number();
```

Comment:
what's difference `Pallet` and `Module`? Just renaming?

Reply:
same question here 👍

Reply:
At around v0.9.1, this is renamed from upstream